webapp/assets/styles/main.scss

```diff
@@ -171,6 +171,8 @@ hr {
 }
 
 .hyphenate-text {
+  -webkit-hyphens: auto;
+  -moz-hyphens: auto;
```

Comment:
we have postcss/autoprefixer doing that for us :wink:

Reply:
I was not sure if it is necessary. So just kick it, or is it better I kick it and push it up again?

Reply:
Ok. I removed it.